### ISSUE TITLE
Pass language options to hlint

### ISSUE THREAD

**Issue by wiz** [atom-haskell/haskell-ghc-mod#110]
`hlint` tries to parse quasiquoted content as haskell code and warns about parsing errors everywhere. It is possible with `-X` option calm it down, but there's no setting to configure this.

Even better, if hlint could receive language options collected from cabal and module automatically.

**Comment by lierdakil** [comment]
Yes. This is mostly a ghc-mod problem. I'm aware of it, but didn't manage to get myself to fix it yet. Workaround for the time being is specifying flags in module header...
